### PR TITLE
Delete f-string in codebase to preserve python 3.5 support.

### LINKED ISSUE #7124: Jupyterlab 1.1.0 not compatible with Python 3.5 due to f-string usage?
## Description

Jupyterlab 1.1.0 hits a `SyntaxError` when run on Python 3.5.2 (Ubuntu 16.04 Python 3 via apt):

```
Traceback (most recent call last):
  File "/usr/local/bin/jupyter-lab", line 6, in <module>
    from jupyterlab.labapp import main
  File "/usr/local/lib/python3.5/dist-packages/jupyterlab/labapp.py", line 22, in <module>
    from .commands import (
  File "/usr/local/lib/python3.5/dist-packages/jupyterlab/commands.py", line 557
    self.logger.info(f'Building jupyterlab assets ({command})')
                                                             ^
SyntaxError: invalid syntax
```
This appears to be due to the use of an f-string in one of the modules, which will only work in Python 3.6+: [commands.py](https://github.com/jupyterlab/jupyterlab/blame/v1.1.0/jupyterlab/commands.py#L557). According to the [changelog](https://jupyterlab.readthedocs.io/en/stable/getting_started/changelog.html#v1-1-0), JupyterLab v1.1.0 should still support 3.5+.

## Context

- Operating System and version: Ubuntu 16.04 LTS
- Browser and version: Firefox 68.0.2
- JupyterLab version: 1.1.0

### PR DESCRIPTION
**Pull request by jasongrout**
## References

Fixes #7124

## Code changes
We currently support python 3.5+, but f-strings come in 3.6.

## User-facing changes

None

## Backwards-incompatible changes

None

### CODE REVIEW

File: jupyterlab/commands.py [hunk 1]
```diff
@@ -554,7 +554,7 @@ def build(self, name=None, version=None, static_url=None,
                         'prod')
         command = ':'.join(parts)
 
-        self.logger.info(f'Building jupyterlab assets ({command})')
+        self.logger.info('Building jupyterlab assets (%s)'%command)
```
Comment (saulshanabrook) [inline jupyterlab/commands.py]:
```suggestion
        self.logger.info('Building jupyterlab assets (%s)' % command)
```

Comment (jasongrout) [reply jupyterlab/commands.py]: Saved for a follow-up PR so we can merge without waiting for another test cycle and release today.

Comment (jasongrout) [reply jupyterlab/commands.py]: https://gitter.im/jupyterlab/jupyterlab?at=5d68634ace6f764db2fee102

**Comment by jasongrout** [comment]
Thanks!

**Pull request merged**